platform/android/app/src/main/kotlin/coop/polypoly/polypod/Authentication.kt

```diff
@@ -46,28 +46,28 @@ class Authentication {
 
         fun authenticate(
             activity: FragmentActivity,
-            newStatus: Boolean = false,
+            showAuthTexts: Boolean = false,
             authComplete: ((Boolean) -> Unit)
         ) {
             val isBiometricEnabled = Preferences.isBiometricEnabled(activity)
             if (!biometricsAvailable(activity) ||
-                (!newStatus && !isBiometricEnabled)
+                (!showAuthTexts && !isBiometricEnabled)
             ) {
                 authComplete(true)
                 return
             }
 
             val title =
-                if (isBiometricEnabled)
-                    activity.getString(R.string.re_auth_prompt_title)
-                else
+                if (showAuthTexts)
```

Comment:
This seems to be the only functional change; `showAuthTexts` seems to be the opposite of `isBiometricEnabled`?

Reply:
it's a logic change. we don't mind whether biometrics are on/off, but whether the authentication is triggered on onboarding/settings/start-pod level. it was the cleaner way to apply this without breaking other stuff around. Counting into account that there's no testing to make us confident enough